transactions/src/test/java/org/ehcache/transactions/xa/internal/XAStoreTest.java

```diff
@@ -1488,6 +1490,52 @@ public void testBulkComputeIfAbsent() throws Exception {
   }
 
   @Test
+  public void testCustomEvictionVeto() throws Exception {
```

Comment:
The only thing that needs verification at this level is that the provided veto is called. Testing the proper handling of veto exists elsewhere.